src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -431,6 +432,8 @@ private static bool GetTls13Support()
                 {
                     client = Registry.GetValue(clientKey, "Enabled", null);
                     server = Registry.GetValue(serverKey, "Enabled", null);
+                    Console.WriteLine($"client {client}");
+                    Console.WriteLine($"client {client}");
```

Comment:
Should be server?

Reply:
Yeah, it served its purpose for getting the CI machine info. Reverting...